hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -230,13 +230,8 @@ public Long getLastResolvedTime() {
 
     @Override
     public String toString() {
-        return "Alert{" +
-                "severity=" + severity +
-                ", status=" + status +
-                ", notes=" + notes +
-                ", lifecycle=" + lifecycle +
-                ", resolvedEvalSets=" + resolvedEvalSets +
-                '}';
+        return "Alert [triggerId=" + getTriggerId() + ", severity=" + severity + ", status=" + status + ", tenantId="
```

Comment:
This change is on purpose ? or just automatically re-create the toString() ?
I remember those fields helped a lot in debugging scenarios (and IIRC this method is invoked on tracing/debugging only contexts).